diff --git a/lib/coreBundle.js b/lib/coreBundle.js
index 8ed8bb5de1b12c7c050191123b22a119298826bb..8b8953a36f2cfecd577b08724e0a185685a09ebb 100644
--- a/lib/coreBundle.js
+++ b/lib/coreBundle.js
@@ -38595,7 +38595,7 @@ var init_crBrowser = __esm({
         return this._crPages().map((crPage) => crPage._page);
       }
       async doCreateNewPage() {
-        const { targetId } = await this._browser._session.send("Target.createTarget", { url: "about:blank", browserContextId: this._browserContextId });
+        const { targetId } = await this._browser._session.send("Target.createTarget", { url: "about:blank", browserContextId: this._browserContextId, background: true, focus: false });
         return this._browser._crPages.get(targetId)._page;
       }
       async doGetCookies(urls) {
