diff --git a/lib/coreBundle.js b/lib/coreBundle.js
index c6d14d3fcf0fe645ee66d5c18834ebd995513e0e..580e865940d7880fee5ca999a049d28f139e2a72 100644
--- a/lib/coreBundle.js
+++ b/lib/coreBundle.js
@@ -39336,7 +39336,11 @@ var init_crBrowser = __esm({
          return this._crPages().map((crPage) => crPage._page);
        }
        async doCreateNewPage() {
 -        const previousPage = this._crPages().at(-1)?._page;
 -        const { targetId } = await this._browser._session.send("Target.createTarget", { url: "about:blank", browserContextId: this._browserContextId, background: true, focus: false });
 -        if (previousPage) await previousPage.bringToFront();
 -        return this._browser._crPages.get(targetId)._page;
 +        const previousPage = this._crPages().at(-1)?._page;
 +        const { targetId } = await this._browser._session.send("Target.createTarget", { url: "about:blank", browserContextId: this._browserContextId, background: true, focus: false });
 +        if (previousPage) await previousPage.bringToFront();
         return this._browser._crPages.get(targetId)._page;
+        const { targetId } = await this._browser._session.send("Target.createTarget", { url: "about:blank", browserContextId: this._browserContextId, background: true, focus: false });
         return this._browser._crPages.get(targetId)._page;
       }
       async doGetCookies(urls) {
